### PR TITLE
fix: Kong for Kubernetes Enterprise (DB Mode)

### PR DESCRIPTION
**Pull request by Rajakavitha1**
### Description

tested, validated, and rewrote the guide.
However, in the cluster the external IP address for LoadBalancer never gets assigned and I was not able to check the Kong Manager UI.
```
rajakavitha@Rajakavithas-MacBook-Air ~ % kubectl get nodes
NAME       STATUS   ROLES                  AGE   VERSION
orbstack   Ready    control-plane,master   16m   v1.27.4+orb1
rajakavitha@Rajakavithas-MacBook-Air ~ % kubectl get services -n kong
NAME                      TYPE           CLUSTER-IP        EXTERNAL-IP    PORT(S)                      AGE
kong-manager              LoadBalancer   192.168.194.233   <pending>      80:30463/TCP                 15m
kong-proxy                LoadBalancer   192.168.194.201   <pending>      80:30885/TCP,443:30841/TCP   15m
kong-validation-webhook   ClusterIP      192.168.194.209   <none>         443/TCP                      15m
postgres                  ClusterIP      192.168.194.231   <none>         5432/TCP                     15m
kong-admin                LoadBalancer   192.168.194.176   198.19.249.2   80:31890/TCP                 15m
rajakavitha@Rajakavithas-MacBook-Air ~ % kubectl get svc -n kong kong-admin --output=jsonpath='{.status.loadBalancer.ingress[0].ip}'
198.19.249.2%                                                                                                          rajakavitha@Rajakavithas-MacBook-Air ~ % KONG_ADMIN_IP=$(kubectl get svc -n kong kong-admin --output=jsonpath='{.status.loadBalancer.ingress[0].ip}')
rajakavitha@Rajakavithas-MacBook-Air ~ % echo $KONG_AMIN_IP

rajakavitha@Rajakavithas-MacBook-Air ~ % export KONG_ADMIN_IP=$(kubectl get svc -n kong kong-admin --output=jsonpath='{.status.loadBalancer.ingress[0].ip}')
rajakavitha@Rajakavithas-MacBook-Air ~ % echo $KONG_AMIN_IP

rajakavitha@Rajakavithas-MacBook-Air ~ % echo $KONG_ADMIN_IP
198.19.249.2
rajakavitha@Rajakavithas-MacBook-Air ~ % kubectl patch deployment -n kong ingress-kong -p "{\"spec\": { \"template\" : { \"spec\" : {\"containers\":[{\"name\":\"proxy\",\"env\": [{ \"name\" : \"KONG_ADMIN_API_URI\", \"value\": \"${KONG_ADMIN_IP}\" }]}]}}}}"
deployment.apps/ingress-kong patched (no change)
rajakavitha@Rajakavithas-MacBook-Air ~ % export PROXY_IP=$(kubectl get -o jsonpath="{.status.loadBalancer.ingress[0].ip}" service -n kong kong-proxy)

rajakavitha@Rajakavithas-MacBook-Air ~ % echo $PROXY_IP

rajakavitha@Rajakavithas-MacBook-Air ~ % kubectl get services -n kong
NAME                      TYPE           CLUSTER-IP        EXTERNAL-IP    PORT(S)                      AGE
kong-manager              LoadBalancer   192.168.194.233   <pending>      80:30463/TCP                 19m
kong-proxy                LoadBalancer   192.168.194.201   <pending>      80:30885/TCP,443:30841/TCP   19m
kong-validation-webhook   ClusterIP      192.168.194.209   <none>         443/TCP                      19m
postgres                  ClusterIP      192.168.194.231   <none>         5432/TCP                     19m
kong-admin                LoadBalancer   192.168.194.176   198.19.249.2   80:31890/TCP                 19m
rajakavitha@Rajakavithas-MacBook-Air ~ % kubectl get services -n kong
NAME                      TYPE           CLUSTER-IP        EXTERNAL-IP    PORT(S)                      AGE
kong-manager              LoadBalancer   192.168.194.233   <pending>      80:30463/TCP                 21m
kong-proxy                LoadBalancer   192.168.194.201   <pending>      80:30885/TCP,443:30841/TCP   21m
kong-validation-webhook   ClusterIP      192.168.194.209   <none>         443/TCP                      21m
postgres                  ClusterIP      192.168.194.231   <none>         5432/TCP                     21m
kong-admin                LoadBalancer   192.168.194.176   198.19.249.2   80:31890/TCP                 21m
rajakavitha@Rajakavithas-MacBook-Air ~ % 
```

### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_src/kic-v2/deployment/kong-enterprise.md [hunk 1]
```diff
@@ -1,122 +1,101 @@
...
+    $ kubectl create namespace kong
+    ```
+    The results should look like this:
+    ```text
+    namespace/kong created
+    ```
+
+* Create Kong Enterprise bootstrap password. Replace `<cloudnative>` with a random password of your choice and note it down
+    ```bash
+    $ kubectl create secret generic kong-enterprise-superuser-password  -n kong --from-literal=password=<cloudnative>
```
Comment (mheap) [inline app/_src/kic-v2/deployment/kong-enterprise.md:22]: Using `<cloudnative>` means that I can't copy/paste when testing. Can we go back to using `cloudnative`?

**Review by mheap — CHANGES_REQUESTED** [review]
I couldn't log in either, but I don't think it's a docs issue. We're going to simplify in KIC 3.0 so I'm happy to merge once the `<>` issue is resolved

File: app/_src/kic-v2/deployment/kong-enterprise.md (hunk 1 above)
Comment (Rajakavitha1) [reply app/_src/kic-v2/deployment/kong-enterprise.md]: Fixed

**Review by mheap — CHANGES_REQUESTED** [review]
Please fix the Vale issues

**Review by mheap — APPROVED** [review]

**Pull request merged**